annotation/src/main/java/org/cbioportal/genome_nexus/annotation/web/AnnotationController.java

```diff
@@ -351,6 +355,25 @@ private IsoformOverride getIsoformOverride(String source, String transcriptId)
 
         return hotspots;
     }
+    
+    @ApiOperation(value = "Perform lookups of Ensembl identifiers and retrieve their external referenes in other databases", 
+            nickname = "getGeneXrefs")
+    @ApiResponses(value = {
+        @ApiResponse(code = 200, message = "Success",
+            response = GeneXref.class,
+            responseContainer = "List"),
+        @ApiResponse(code = 400, message = "Bad Request")
+    })
+    @RequestMapping(value = "/xrefs/{accession}", 
+            method = RequestMethod.GET, 
+            produces = "application/json")
+    public List<GeneXref> getGeneXrefs(
+            @PathVariable
+            @ApiParam(value="Ensembl gene accession. For example ENSG00000169083",
```

Comment:
@sheridancbio I wanted to keep the styling consistent with the rest of  `AnnotationController` (example [here](https://github.com/cBioPortal/genome-nexus/blob/master/annotation/src/main/java/org/cbioportal/genome_nexus/annotation/web/AnnotationController.java#L99))